src/upgrade.js

```diff
@@ -314,6 +314,44 @@ Upgrade.upgrade = function (callback) {
 				next();
 			}
 		},
+		function (next) {
+			thisSchemaDate = Date.UTC(2017, 1, 22);
+			var schemaName = '[2017/2/22] Fixing urls in config';
+
+			if (schemaDate < thisSchemaDate) {
+				updatesMade = true;
+				winston.info(schemaName);
+				async.waterfall([
+					function (cb) {
+						db.getObject('config', cb);
+					},
+					function (config, cb) {
+						if (!config) {
+							return cb();
+						}
+
+						Object.keys(config).forEach(function (key) {
+							var oldValue = config[key];
+
+							if (typeof oldValue !== 'string') {
+								return;
+							}
+						
+							config[key] = oldValue.replace(/(\/assets)?\/images\//g, '/assets/images/');
```

Comment:
This blindly replaces every string-y config value with `/assets/images` if the string matches, and while it works fine, I'd feel safer if you iterated through a list of known config values that need updating. Just in case.

Extremely unlikely edge case, if someone's site description was `Collection of memes/images` would be incorrectly updated to `Collection of memes/assets/images`